src/StationClass.js

```diff
@@ -0,0 +1,15 @@
+const { bookshelf, Passenger, Passengers, knex } = require('../database/connection.js')
+const commands = require('./commands.js')
+
+class StationClass {
+
+  getById(id){
+    return commands.getStation( {id} )
+  }
+
+  getByName(Name){
```

Comment:
convention is to use lower, camel case for variable identifiers (`Name` looks like a class to me)